h5p-row.js

```diff
@@ -30,12 +30,20 @@ H5P.Row = (function (EventDispatcher) {
     var createHTML = function () {
       // Create wrapper
       wrapper = document.createElement('div');
+      wrapper.classList.add('h5p-row');
+
+      for (var i = 0; i < params.columns.length; i++) {
+        var column = document.createElement('div');
```

Comment:
This should recieve some inline styling to fit the width and paddings that come with the forked column:

<img width="562" alt="image" src="https://github.com/h5p/h5p-row/assets/42533113/5f1f20e6-4073-4fe2-8c67-b9079b7d6037">